tests/Fixtures/Integration/set/@Symfony.test-in.php

```diff
@@ -65,7 +65,7 @@ private function transformText($dummy, array $options = array(),$data=null)
             return ucwords($dummy);
         }
 
-        throw new \RuntimeException(sprintf('Unrecognized dummy option "%s"', $dummy));
+        throw new \RuntimeException(sprintf('Unrecognized dummy option "%s".', $dummy));
```

Comment:
Is this change actually related to the new option?

Reply:
It is not, but fabbot.io starts [nudging](https://fabbot.io/report/FriendsOfPHP/PHP-CS-Fixer/5757/34120b69992a615c906195243a6fda1143354343) about those files once I touch them. 

What should I do about it?

Reply:
It happens sometimes, I'd say we can ignore that.